seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-jdbc/src/main/scala/org/apache/seatunnel/spark/jdbc/sink/Jdbc.scala

```diff
@@ -65,6 +66,7 @@ class Jdbc extends SparkBatchSink {
         "saveMode" -> "error",
         "useSsl" -> "false",
         "showSql" -> "true",
+        "isolationLevel" -> "READ_UNCOMMITTED",
```

Comment:
I think the default value should be `READ_COMMITTED`, and please add this config to document

Reply:
In Spark ，the default value of isolationLevel is  "READ_UNCOMMITTED". Please see https://spark.apache.org/docs/latest/sql-data-sources-jdbc.html.

Reply:
OK. I will add this config to document soon.